fast64_internal/oot/oot_level_writer.py

```diff
@@ -1003,63 +1001,103 @@ def draw(self, context):
         layout.label(text="Are you sure you want to remove this scene?")
 
 
+def drawSceneSearchOp(layout, enumValue, opName):
+    searchBox = layout.box().row()
+    searchBox.operator(OOT_SearchSceneEnumOperator.bl_idname, icon="VIEWZOOM", text="").opName = opName
+    searchBox.label(text=getEnumName(ootEnumSceneID, enumValue))
+
+
 class OOT_ExportScenePanel(OOT_Panel):
     bl_idname = "OOT_PT_export_level"
     bl_label = "OOT Scene Exporter"
 
-    def drawSceneSearchOp(self, layout, context, enumValue, opName):
-        searchBox = layout.box().row()
-        searchBox.operator(OOT_SearchSceneEnumOperator.bl_idname, icon="VIEWZOOM", text="").opName = opName
-        searchBox.label(text=getEnumName(ootEnumSceneID, enumValue))
-
     def draw(self, context):
         col = self.layout.column()
-        exportOp: OOT_ExportScene = col.operator(OOT_ExportScene.bl_idname)
-        # if not bpy.context.scene.ignoreTextureRestrictions:
-        # 	col.prop(context.scene, 'saveTextures')
+
         settings: OOTExportSceneSettingsProperty = context.scene.ootSceneExportSettings
         if settings.customExport:
             prop_split(col, settings, "exportPath", "Directory")
             prop_split(col, settings, "name", "Name")
             customExportWarning(col)
         else:
-            self.drawSceneSearchOp(col, context, settings.option, "Export")
+            drawSceneSearchOp(col, settings.option, "Export")
             if settings.option == "Custom":
                 prop_split(col, settings, "subFolder", "Subfolder")
                 prop_split(col, settings, "name", "Name")
 
         prop_split(col, context.scene, "ootSceneExportObj", "Scene Object")
 
+        col.prop(settings, "singleFile")
+        col.prop(settings, "customExport")
+
         if context.scene.fast64.oot.hackerFeaturesEnabled:
+            hackerOoTBox = col.box()
```

Comment:
```suggestion
            hackerOoTBox = col.box().column()
```
Don't use the results of box() directly, this causes big spaces between lines.

Reply:
good to know